### PR TITLE
Adding in process entropy query

### PR DESCRIPTION
**Pull request by shainw**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Hunting Queries/SecurityEvent/ProcessEntropy.txt [hunk 1]
```diff
@@ -14,6 +14,7 @@
 //
 // Tactics: #Execution
 //
+// May need to reduce the number of days if the environment is very large.  Try 3-5 days, but less accuracy.
 let end = startofday(ago(7d));
```
Comment (ianhelle) [inline Hunting Queries/SecurityEvent/ProcessEntropy.txt]: Shouldn't this be startofday(now()) ?
Or is it really meant to be the 7 days before the current 7 days?

Comment (shainw) [reply Hunting Queries/SecurityEvent/ProcessEntropy.txt]: Yes...typo.  Thanks for catching. :)

File: Hunting Queries/SecurityEvent/ProcessEntropy.txt [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+| summarize count() by Process 
+// Removing noisy processes for an environment, adjust as needed
+| where count_ >= 100000;
+let AllSecEvents = SecurityEvent
+| where TimeGenerated >= start and TimeGenerated <= end
+| where EventID == 4688
+| where Process !in~ ("conhost.exe")
+| project Computer, Process;
+// Removing noisy process from full list
+let Include = AllSecEvents | join kind= anti (
```
Comment (timbMSFT) [inline Hunting Queries/SecurityEvent/ProcessEntropy.txt]: Put smaller table on left hand side of join for better performance (and use leftanti/rightanti as needed

File: Hunting Queries/SecurityEvent/ProcessEntropy.txt [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+Exclude
+) on Process;
+// Identifying prevalence for a given process in the environment
+let DCwPC = Include
+| join kind=inner (
+Include | summarize DistinctComputersWithProcessCount = dcount(Computer) by Process
```
Comment (timbMSFT) [inline Hunting Queries/SecurityEvent/ProcessEntropy.txt]: ditto - the summarized version of the table will always be smaller -- so put this on left hand side of the join (Include | summarize... by Process) | join kind=inner Include

File: Hunting Queries/SecurityEvent/ProcessEntropy.txt [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+) on Process
+| distinct Computer, Process, DistinctComputersWithProcessCount;
+// Getting the Total process count on each host to use as the denominator in the entropy calc
+let TPCoH = Include
+| join kind=inner (
+Include | summarize TotalProcessCountOnHost = count(Process) by Computer //ProcTotalDistCountOnHost is my Classes
```
Comment (timbMSFT) [inline Hunting Queries/SecurityEvent/ProcessEntropy.txt]: ditto

File: Hunting Queries/SecurityEvent/ProcessEntropy.txt (hunk 2 above)
Comment (shainw) [reply Hunting Queries/SecurityEvent/ProcessEntropy.txt]: Done.

File: Hunting Queries/SecurityEvent/ProcessEntropy.txt (hunk 3 above)
Comment (shainw) [reply Hunting Queries/SecurityEvent/ProcessEntropy.txt]: Done

File: Hunting Queries/SecurityEvent/ProcessEntropy.txt (hunk 4 above)
Comment (shainw) [reply Hunting Queries/SecurityEvent/ProcessEntropy.txt]: Done

**Pull request merged**